deployment/hasura/migrations/Aerie/3_sequence_parcel/down.sql

```diff
@@ -0,0 +1,203 @@
+
+------------------------------------
+--- Modify Expanded Sequences Table
+------------------------------------
+alter table sequencing.expanded_sequences
+  add column edsl_string text;
+
+update sequencing.expanded_sequences
+set edsl_string = '{}'
+where edsl_string is null;
+
+alter table sequencing.expanded_sequences
+  alter column edsl_string set not null;
+
+------------------------------------------
+---- Modify the Command Dictionary Table
+------------------------------------------
+
+alter table sequencing.command_dictionary
+  add column command_types_typescript_path text;
+
+--- Data Migration
+update sequencing.command_dictionary
+set command_types_typescript_path = dictionary_path
+where command_types_typescript_path is null;
+
+
+alter table sequencing.command_dictionary
+  alter column command_types_typescript_path set not null;
+
+---
+
+drop trigger set_timestamp on sequencing.command_dictionary;
+
+comment on column sequencing.command_dictionary.command_types_typescript_path is e''
+  'The location of command dictionary types (.ts) on the filesystem';
+
+alter table sequencing.command_dictionary
+  drop column updated_at,
+  drop column dictionary_path;
```

Comment:
This part of the migration can be simplified, since this is just a column rename and drop:

```sql
------------------------------------------
---- Modify the Command Dictionary Table
------------------------------------------
drop trigger set_timestamp on sequencing.command_dictionary;

alter table sequencing.command_dictionary
  rename column dictionary_path to command_types_typescript_path;

alter table sequencing.command_dictionary
  drop column updated_at;
```

Reply:
with this change do I have to create a comment still:

```
comment on column sequencing.command_dictionary.command_types_typescript_path is e''
  'The location of command dictionary types (.ts) on the filesystem';
```

Reply:
Nope! The text is the same, so it'll get auto-adjusted to the new column name.

Reply:
Awesome